examples/data_pipeline/pipelines.py

```diff
@@ -24,7 +24,7 @@ def build_sp_pipeline(spm_file):
 
     # Insert token in vocab to match a pretrained vocab
     vocab.insert_token('<pad>', 1)
-    pipeline = TextSequentialTransforms(tokenizer, vocab, ToLongTensor())
+    pipeline = TextSequentialTransforms(tokenizer, vocab)
```

Comment:
Why are we removing the call to `ToLongTensor `?

Reply:
Because right now, `vocab` object returns a list of token list with different length. So when it calls `ToLongTensor`, it will fail. After I land padding func https://github.com/pytorch/text/pull/952, I will add padding func together with `ToLongTensor`.

Reply:
Okay that sounds good. 

NIT: might be worth adding a TODO comment just to keep track of this